src/styles/index.ts

```diff
@@ -1331,6 +1331,18 @@ const styles = (theme: ThemeColors) =>
             lineHeight: variables.lineHeightNormal,
         },
 
+        textSupportingSmallSize: {
+            fontFamily: FontUtils.fontFamily.platform.EXP_NEUE,
+            fontSize: variables.fontSizeSmall,
+            color: theme.textSupporting,
+        },
+
+        textSupportingNormalSize: {
```

Comment:
Are we sure this doesn't exist already somewhere? I feel like we use this style a lot, so I would imagine there is something we can reuse.

Reply:
We have similar styles in code, but I haven't found any for this particular fontSize.

Reply:
Well I'm saying, just supplement your normal sized style with `colorMuted`, so basically give it two classes, if that makes sense. Since `colorMuted` already exists.

Reply:
@WojtekBoman actually you can pull from latest main, i remember resolving this